js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -446,6 +451,13 @@ export class OpenAIModel implements PromptCompletionModel {
         }
         params.model = model;
 
+        // Remove tool params if not using tools
+        if (!Array.isArray(params.tools) || params.tools.length == 0) {
+            if (params.tool_choice) {
+                delete params.tool_choice;
+            }           
+        }
+
```

Comment:
Not sure this is necessary once Tools Augmentation changes are merged in:
https://github.com/microsoft/teams-ai/pull/1916/files#diff-15e68abdb314274fe327871cd9251953b2946714792b5cd3c401d33b687a9d92R512-R541

Reply:
It'll still be needed... It's a quirk in their API. They return a 400 error if you include this param (even if set to "auto") and don't provide tools. We're always setting it to "auto" regardless of the augmentation so it has to be removed programmatically